features/step_definitions/operators.rb

```diff
@@ -40,6 +40,22 @@
   end
 end
 
+Given /^I create a new OperatorSource$/ do
+  ensure_admin_tagged
+  # Create OperatorSource in 4.6-
+  if env.version_lt("4.6", user: user)
```

Comment:
I think it makes more sense to check whether this already exist and create if not. What happens if two tests try to create?
It appears to me as a destructive test because it creates a cluster global resource. Then what cleans this resource?